### PR TITLE
Move GetStoragePoolVolume to ClusterTx

### PR DESCRIPTION
**Pull request by monstermunchkin**

### CODE REVIEW

File: lxd/db/entity.go [hunk 1]
```diff
@@ -320,7 +320,7 @@ func (c *Cluster) GetURIFromEntity(entityType int, entityID int) (string, error)
 			return "", fmt.Errorf("Failed to get storage volume: %w", err)
 		}
 
-		uri = fmt.Sprintf(cluster.EntityURIs[entityType], instance.PoolName, instance.Name, backup.Name, instance.ProjectName)
+		uri = fmt.Sprintf(cluster.EntityURIs[entityType], volume.PoolName, volume.TypeName, volume.Name, backup.Name, volume.ProjectName)
```
Comment (tomponline) [inline lxd/db/entity.go:323]: good catch!

**Comment by tomponline** [comment]
Could you include ab40e12d5bc2c44e42d30a9f94174c043091f769 in this PR too?

**Review by tomponline — APPROVED** [review]
LGTM thanks!

**Comment by monstermunchkin** [comment]
@tomponline this should allow for a single transaction when deleting expired storage volume backups.

**Comment by tomponline** [comment]
Dead code check failed

**Comment by monstermunchkin** [comment]
> Dead code check failed 

Fixed.

**Review by tomponline — APPROVED** [review]

**Pull request merged**